sdk/eventhub/event-hubs/src/eventHubSender.ts

```diff
@@ -393,7 +384,8 @@ export class EventHubSender extends LinkEntity {
         );
         await defaultLock.acquire(this.senderLock, () => {
           const config: RetryConfig<void> = {
-            operation: () => this._init(),
+            operation: () =>
+              this._init(this._createSenderOptions(Constants.defaultOperationTimeoutInMs)),
```

Comment:
For ease of readability, consider not inlining the call to `this._createSenderOptions` here and use a constant `senderOptions` that can store the output of `this._createSenderOptions`